[Private preview] Add docs for Cloud Connect + AutoOps for self-managed clusters (#2287)
elastic/docs-content: deploy-manage/monitor/autoops/cc-manage-connected-clusters.md

---
applies_to:
  deployment:

>     self:

Reviewer: ```suggestion
    self:
    eck:
    ece:
```

navigation_title: Manage connected clusters
---

# Manage connected clusters

> Once your self-managed clusters are connected to {{ecloud}}, you can manage them from your {{ecloud}} home page.

Reviewer 2: ```suggestion
After your self-managed clusters have connected to {{ecloud}}, you can manage them from your {{ecloud}} home page.
```

:::{important}
You need the **Organization owner** role to perform these actions.
:::

## Connect additional clusters

To connect more self-managed clusters, we recommend repeating the steps to [connect your self-managed cluster to AutoOps](/deploy-manage/monitor/autoops/cc-connect-self-managed-to-autoops.md).

> You can use the same installation command to connect multiple clusters, but remember that each cluster needs a separate, dedicated Elastic Agent.

Author: You're right. I had originally mentioned it above, but we ended up taking out that section. Fixed
